packages/usa-accordion/src/styles/_usa-accordion.scss

```diff
@@ -42,7 +69,7 @@ $accordion-border: units($theme-accordion-border-width) solid
 // scss-lint:enable PropertyCount
```

Comment:
We might want to remove this `forced-colors-adjust: none`. At the time it helped to make sure that the accordion buttons didn't disappear on high contrast mode, but with the color override it keeps the defined colors. 

![Screenshot 2023-05-04 at 11 32 09 AM](https://user-images.githubusercontent.com/25211650/236256406-7d8fde9d-15be-4f94-9b58-79262ec85a3d.png)

![Screenshot 2023-05-04 at 10 35 13 AM](https://user-images.githubusercontent.com/25211650/236249105-6b89597a-ce02-4b3a-8233-1be7d96dd97a.png)

I haven't worked out the perfect solution yet but maybe we could set high contrast token colors and use `set-text-and-bg`.

```scss
@media (forced-colors: active) {
  @include set-text-and-bg(
    $accordion-high-contrast
    $context: $accordion-context
  );
}
```

Let me know if you want this to be a part of this solution, or if we want to ticket the high contrast work out separately

Reply:
@mahoneycm the first screenshot looks better for high contrast mode. Is there a way I can set that as the default for HCM?

Reply:
I created and completed the work to resolve the HCM display: #5286 

This work can be merged separately to `develop`, or I can PR it against this branch instead 👍